dart/lib/src/protocol/sentry_user.dart

```diff
@@ -37,13 +39,21 @@ class SentryUser {
     this.email,
     this.ipAddress,
     this.segment,
-    Map<String, dynamic>? extras,
-  })  : assert(id != null ||
-            username != null ||
-            email != null ||
-            ipAddress != null ||
-            segment != null),
-        extras = extras == null ? null : Map.from(extras);
+    this.subscription,
```

Comment:
`subscription` is part of the JSON example, but not part of the spec https://develop.sentry.dev/sdk/event-payloads/types/#user
I have to confirm this.

Reply:
Let's remove `subscription`.
It's not part of the protocol, users can set it into the `data` field.